tests/tests/cpld_bmark_p8

```diff
@@ -63,18 +64,23 @@ export FHMAX=6
 export RESTART_N=`expr ${FHMAX} - ${FHROT}`
 export RESTART_INTERVAL="${RESTART_N} -1"
 
-export TASKS=$TASKS_cpl_bmrk
-export TPN=$TPN_cpl_bmrk
-export INPES=$INPES_cpl_bmrk
-export JNPES=$JNPES_cpl_bmrk
-export THRD=$THRD_cpl_bmrk
-export WRTTASK_PER_GROUP=$WPG_cpl_bmrk
+export WLCLK=45
```

Comment:
Does this test require 45 minutes?

Reply:
It was consistently running out of time during testing.  I can check to see what the time is now though.

Reply:
Orion wall time is:  1098.278576 
I'll make sure the other machines have similar timing and then we can remove the WLCLCK.